docs/1___overview.adoc

```diff
@@ -124,6 +124,9 @@ In some ways the Scheduled Execution interface has similarities to the Model Exc
 [#figure-scheduled-execution-data-flow]
 image::images/scheduled-execution-data-flow.svg[width=40%, align="center"]
 
+_[An external scheduler is only required if the importer needs direct control over the tasks, e.g. to avoid overruns in a real-time application.
```

Comment:
I don't like the "is only required" because this is not relevant perspective. Better "An external scheduler allows to control scheduling of tasks and thus model partitions. Such an environment e.g. is required to execute real-time simulations." 

If you really need to reference _overruns_ I suggest "Only in a real-time execution overruns can occur and thus can be stressed." Actually mentioning _preemption_ and _priorities_ would be just as relevant to mention. So I rather suggest not to mention any since it is explained in more detail in the SE section.